Tools/rollup-plugin-strip-pragma/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "rollup-plugin-strip-pragma",
+  "version": "1.0.0",
+  "description": "A Rollup plugin to strip requirejs build pragmas from your code.",
+  "homepage": "http://cesiumjs.org",
```

Comment:
Change to cesium.com

Reply:
The homepage for open source CesiumJS related things is still ceisumjs.org, will get changed in the future if/when we do any kind of switchover.